src/bootstrap.js

```diff
@@ -129,6 +165,29 @@ async function logProgress(offset, nbDocs) {
     Math.round(ratePerSecond),
     ms(((totalDocs - offset - nbDocs) / ratePerSecond) * 1000)
   );
+
+  // ------- Last minutes
+  const thisMinute = new Date().getMinutes();
```

Comment:
not sure if I understand it correctly, but this logic might be wrong across day limits? Might be better to work with unix seconds/ ms

Reply:
if too tricky, I'm fine with using `ms`  or date-fns for this

Reply:
`.getMinutes()` returns 0-59.

`thisMinute` is stored on a array `lastMinutes`, like this

```javascript
lastMinutes = [
  [ <minute>, <count> ]
];
```

this enable us to reuse the key "minute" to and have only N entry for N minutes. where N = 10.
**E.g:**
```javascript
lastMinutes = [
  [ 7, <count> ]
  [ 8, <count> ]
  [ 9, <count> ]
  [ 10, <count> ]
  [ 11, <count> ]
  [...]
];
```

at line 171 we shift the array if the length is > higher than N (10), we can not have more than 10 entries so there is never an overlap between hours or days. This we way have rolling stats over the last 10minutes easily.